cd/helm/nef/templates/mongo-express-nef-route.yaml

```diff
@@ -0,0 +1,30 @@
+apiVersion: route.openshift.io/v1
+kind: Route
+metadata:
+  creationTimestamp: "2022-09-15T19:03:41Z"
+  labels:
+    app.kubernetes.io/managed-by: Helm
+    io.kompose.service: mongo-express-nef
+  name: mongo-express-nef
+  namespace: evol5-capif
+  resourceVersion: "753695354"
+  uid: 1cf39ca5-e0bc-47e7-8591-7e173f3b1110
+spec:
+  host: mongo-express-nef.apps.ocp-epg.hi.inet
+  port:
+    targetPort: 8081
```

Comment:
Adapt name according to mongo-express-nef-service port name definition. Avoid use number